OCM/tf/ocm_tf_checker.cc

```diff
@@ -1696,6 +1696,13 @@ static bool IsOpInputDimZeroTF(tensorflow::Node *node, int * ov_version) {
   return is_input_dim_zero;
 }
 
+static Status CheckIfOutputNode(const Node* node,
+                                const std::set<std::string> skip_these_nodes,
+                                bool& skip_it) {
+  skip_it = !(skip_these_nodes.find(node->name()) != skip_these_nodes.end());

```

Comment:
Though the logic is finem but for simplification, can these 2 negations be replaced with a single "==" comparison inside the node find check? And then we can add one extra line to relate "is_node_supported" and "skip_it"